RadarSDK/RadarSettings.m

```diff
@@ -120,8 +139,13 @@ + (void)setConfig:(NSDictionary *)config {
 }
 
 + (RadarLogLevel)logLevel {
-    NSInteger logLevelInteger = [[NSUserDefaults standardUserDefaults] integerForKey:kLogLevel];
-    return (RadarLogLevel)logLevelInteger;
+    RadarLogLevel logLevel;
+    if ([[NSUserDefaults standardUserDefaults] objectForKey:kLogLevel] == nil) {
+        logLevel = RadarLogLevelInfo;
```

Comment:
this makes the default log level info?  Is there any risk of having this on in production, ie taking up space on the device in system logs?

Reply:
Nope, should be fine. The only info-level logs are for locations/events/errors (relatively low volume), whereas all other logs are debug-level. We could add a check for environment (`publishableKey` contains test) but I think it's overkill for v1

Reply:
Does the device delete these every so often of they do they take up space on the device forever?

Reply:
The OS handles it. There's a max buffer size. On Android, I believe the buffer is 4 MB by default. On iOS, not sure it's documented (we call `NSLog`, see https://developer.apple.com/documentation/foundation/1395074-nslogv). iOS apps print a ton of logs in production, so these will be like a drop of water in a 10 gallon bucket. Not worried about it. We'll also document `Radar.setLogLevel()` in case people want to turn this off.